src/OpenApi/Schemas/Models/TagSchema.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace CSlant\Blog\Api\OpenApi\Schemas\Models;
+
+use OpenApi\Attributes\Property;
+use OpenApi\Attributes\Schema;
+
+#[Schema(
+    schema: "Tag",
+    required: ["id", "name", "slug", "description", "status", "author_id", "author_type"],
```

Comment:
check some fields can be null in the database and we'll expect it not required.

<img width="767" alt="image" src="https://github.com/user-attachments/assets/a54bc436-4a75-4afb-8b66-05ece10e808b" />